packages/augur-sdk/src/state/getter/Trading.ts

```diff
@@ -586,33 +584,19 @@ export class Trading {
   }
 }
 
+// TODO: Review if we could specify _desired_ reporting states instead. $not cannot make use of indexes
```

Comment:
From the UI perspective: Yes, this can be inclusive instead of exclusive filter.

Reply:
Does that make this todo moot?

Reply:
No it means we should alter this and also alter and calling instances to instead pass a list of inclusive reporting states instead of excluded reporting states